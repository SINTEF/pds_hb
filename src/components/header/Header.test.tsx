import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from '.'

it('renders without crashing', () => {
  const tree = renderer
    .create(
      <Header
        isCompanyUser={true}
        username="Navn Navnesen"
        onClick={() => {
          return
        }}
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
