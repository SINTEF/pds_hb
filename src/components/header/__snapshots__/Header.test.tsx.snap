// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="home"
    onClick={[Function]}
  >
    PDS Datahandbook
  </div>
  <div>
    <div
      className="title notClicked"
    >
      <div
        className="usersymbol"
        onClick={[Function]}
      >
        N
      </div>
    </div>
  </div>
</div>
`;
